build/common.sh

```diff
@@ -833,3 +833,43 @@ function kube::release::gcs::publish_latest() {
   echo "+++ gsutil cat ${latest_file_dst}:"
   gsutil cat ${latest_file_dst}
 }
+
+# Publish a new latest.txt, but only if the release we're dealing with
+# is newer than the contents in GCS.
+function kube::release::gcs::publish_latest_official() {
+  local new_version=${KUBE_GCS_LATEST_CONTENTS}
+  local latest_file_dst="gs://${KUBE_GCS_RELEASE_BUCKET}/${KUBE_GCS_LATEST_FILE}"
+
+  local version_regex="^v(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)$"
```

Comment:
Pretty nitpicky, but you could use `local -r` to make this a read only variable and catch errors trying to redefine it inside this function. Equivalent of C++'s "const", though probably not important enough to bother with it here... :-)